### ISSUE TITLE
ESlint false positive with custom component.

### ISSUE THREAD

**Issue by ptrxyz** [sveltejs/eslint-plugin-svelte3#194]
ESlint somehow reports an `unsafe-member-access` error when I try to access a exported property of a componente. Consider this component:

```ts
<script lang="ts">
    // comp.svelte
    export const value1 = 3
    export const api = {
        value2: 3,
        get value3() {
            return 1
        },
    }
</script>
```

which is imported here:

```ts
<script lang="ts">
    // +page.svelte
    import Comp from 'comp.svelte'

    let x: Comp

    $: if (x) console.log(x.value1, x.api.value2, x.api.value3, x.something)
</script>

<Comp bind:this={x} />
```

If I run `svelte-kit sync; pnpm lint` I get an error:

```sh
/tmp/eslint-test/src/routes/+page.svelte
  6:37  error  Unsafe member access .value2 on an `any` value  @typescript-eslint/no-unsafe-member-access
  6:51  error  Unsafe member access .value3 on an `any` value  @typescript-eslint/no-unsafe-member-access
```

However `api` _is_ actually typed. Typescript also doesn't seem to complain:

![image](https://user-images.githubusercontent.com/209056/199846841-3c93b57e-5f1f-43a7-9538-b9f74399c63f.png)

 
Does anyone know what's going wrong here?

--
A minimal example based on the SvelteKit template can be found here:
[Github-Repo](https://github.com/ptrxyz/eslint-unsafe-member-example)
[StackBlitz](https://stackblitz.com/github/ptrxyz/eslint-unsafe-member-example?file=src%2Froutes%2F%2Bpage.svelte)
